site/docs/transform/flatten.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: docs
+menu: docs
+title: Flatten
+permalink: /docs/flatten.html
+---
+
+The **flatten** transform maps array-valued _fields_ to a set of individual data objects, one per array entry. This transform generates a new data stream in which each data object consists of an extracted array value as well as all the original fields of the corresponding input data object.
+
+
+## Flatten Transform Definition
+
+{% include table.html props="flatten,as" source="FlattenTransform" %}
+
+
```

Comment:
Having a usage section without specs like Vega does is helpful.  

The visual example is good additions, but having explicit examples how data is reshaped is useful.